src/pt/mangastop/src/eu/kanade/tachiyomi/extension/pt/mangastop/MangaStop.kt

```diff
@@ -18,4 +22,17 @@ class MangaStop : MangaThemesia(
     override val client = super.client.newBuilder()
         .rateLimit(3)
         .build()
+
+    override fun pageListParse(document: Document): List<Page> {
+        val pages = super.pageListParse(document)
+            .filterNot { it.imageUrl?.contains("mihon", true) == true }
+
+        if (pages.isNotEmpty()) return pages
+
+        return MangaThemesia.JSON_IMAGE_LIST_REGEX.find(document.toString())
```

Comment:
Maybe use `override fun pageListParse(response: Response)` instead, to avoid back-and-forth with `document`.